struct StructA {
  1: bool fieldA
  2: i32 fieldB
  3: string fieldC
  4: list<i32> fieldD
  5: set<i32> fieldE
  6: map<i32, string> fieldF
  7: list<list<list<i32>>> fieldG
  8: set<set<set<i32>>> fieldH
  9: map<map<i32, string>, map<i32, string>> fieldI
  10: map<list<set<i32>>, set<list<i32>>> fieldJ
}
